src/script/api/script_base.cpp

```diff
@@ -10,36 +10,46 @@
 #include "../../stdafx.h"
 #include "script_base.hpp"
 #include "script_error.hpp"
+#include "script_log.hpp"
 
 #include "../../safeguards.h"
 
-/* static */ uint32 ScriptBase::Rand()
+/* static */ SQInteger ScriptBase::Rand()
```

Comment:
Isn't it better to use int64 than SQInteger? I always thought SQInteger was specifically used outside the API.

Reply:
`SQInteger` is the type used in every squirrel internal functions (especially `sq_pushinteger()` to pass a number to the squirrel VM and `sq_getinteger()` to get a number from the VM). It is indeed an int64, but I think using SQInteger is better as it hides implementation details from the API.